### ISSUE TITLE
Weights

### ISSUE THREAD

**Issue by muhammads97** [BrandonSmithJ/MDN#12]
Hello, I'm trying to run the code, I tried MSI data and HICO data but it always says that the weights are not found, although I can see that the repo includes zip files for the weights, can you please provide more information on this? or where can I find the weights?

I've also tried to run the training on MSI data, I struggled to find out the structure of the data directory:
data/loc/MSI/Rrs.csv, data/loc/MSI/Rrs_wvl.csv, data/loc/Chl.csv, and the training started, but the logs doesn't provide any information on how manny epochs are done, and when will it finish, or the accuracy, maybe you can provide more details about the training process?

Thank you

**Comment by BrandonSmithJ** [comment]
This may be an issue related to [Git LFS](https://git-lfs.github.com/). Ensure you have it installed, and try cloning the repository again. I also added a check a couple weeks ago that should signal if this is indeed the issue, which would appear as the weight zips being in the folder but only containing the git reference rather than the actual zip. 

As far as running on your own data, there should be logs printed to the console when training (though you might also set --verbose to see additional logs). As far as the process, it uses the standard tensorflow/keras model training API, and you can see the code performing this [here](https://github.com/BrandonSmithJ/MDN/blob/9e161aef5c076d2adce63c4d789da7c96506e19e/model/MDN.py#L285).
